### PR TITLE
Add tests for empty payload consumer commit

### PR DESCRIPTION
**Pull request by matyaskuti**
# About this change - What it does

Following up on #833, adding tests around the fix that was hastily implemented there.

### CODE REVIEW

File: tests/integration/test_rest_consumer.py [hunk 1]
```diff
@@ -251,6 +251,80 @@ async def test_offsets(rest_async_client, admin_client, trail):
...
+        group_name,
+        fmt=fmt,
+        trail=trail,
+        # By default this is true
+        payload_override={"auto.commit.enable": "false"},
+    )
+    topic_name = new_topic(admin_client)
+    offsets_path = f"/consumers/{group_name}/instances/{instance_id}/offsets{trail}"
+    assign_path = f"/consumers/{group_name}/instances/{instance_id}/assignments{trail}"
+    consume_path = f"/consumers/{group_name}/instances/{instance_id}/records{trail}?timeout=5000"
```
Comment (eliax1996) [inline tests/integration/test_rest_consumer.py:270]: Why the optional trailing?

File: tests/integration/test_rest_consumer.py [hunk 2]
```diff
@@ -251,6 +251,80 @@ async def test_offsets(rest_async_client, admin_client, trail):
     assert "partition" in data and data["partition"] == 0, f"Unexpected partition {data}"
 
 
+@pytest.mark.parametrize("trail", ["", "/"])
```
Comment (eliax1996) [inline tests/integration/test_rest_consumer.py:254]: I cannot see the value of that parameter

**Review by eliax1996 — CHANGES_REQUESTED** [review]
Looks fine except for the trail that I feel it’s useless. It is?

File: tests/integration/test_rest_consumer.py (hunk 2 above)
Comment (matyaskuti) [reply tests/integration/test_rest_consumer.py:254]: I've followed the pattern of the other tests here, which all have it. I sort of understand why that is there, with some frameworks this can go wrong fairly easy :(

Comment (aiven-anton) [reply tests/integration/test_rest_consumer.py:254]: I guess what's odd is that we support both, but since we do, I guess it makes sense to test it.

**Review by eliax1996 — APPROVED** [review]
See you spaceboy 🔫
